test/Succeed/Generalize.warn

```diff
@@ -1,3 +1,15 @@
+Generalize.agda:129,14-15: warning: -W[no]HiddenGeneralize
+Declaring a variable as hidden has no effect in a variable block.
+Generalization never introduces visible arguments.
+
+Generalize.agda:129,16-17: warning: -W[no]HiddenGeneralize
+Declaring a variable as hidden has no effect in a variable block.
+Generalization never introduces visible arguments.
+
+Generalize.agda:129,18-19: warning: -W[no]HiddenGeneralize
+Declaring a variable as hidden has no effect in a variable block.
+Generalization never introduces visible arguments.
```

Comment:
An example where a warning at line 129 comes before one on 64 (textual rather than numerical ordering).

Reply:
@plt-amy offered a valid explanation: wrong ranged for Nificier warnings.